### DISCUSSION TITLE
Linking brew PHP and PHPMyAdmin to OS Monterey built in Apache

### DISCUSSION THREAD

**Discussion by jamescoyle1** [Homebrew/discussions#3125] [Getting started]
Is there a way to use Homebrew’s PHP and PHPMyAdmin in conjunction with Mac OS Monterey’s built-in version of Apache2? 

I’d prefer to not need to install Homebrew’s version of Apache2 and need to rebuild the Apache server. Thanks

**Comment by SMillerDev** [comment]
No, this is not supported. There's not much to rebuild though, `cp -R /etc/httpd/* $(brew --prefix)/etc/httpd/` should be enough to copy all configuration.

**Reply by jamescoyle1** [reply]
May I assume then, that this article is erroneous?  It seems to imply that Homebrew's PHP can be used with the OS Monterey built-in version of Apache.

Thanks

https://wpbeaches.com/updating-to-php-versions-7-4-and-8-on-macos-12-monterey/

**Reply by SMillerDev** [reply]
That's not working with the PHP in homebrew/core but with a separate tap: https://github.com/shivammathur/homebrew-php

**Comment by jamescoyle1** [comment]
Thanks. I’ll uninstall PHP and either go to the tap, or install the Homebrew version of Apache and then reinstall PHP. This cleared up a lot of confusion. Thanks again.
